LayoutTests/accessibility/button-in-deep-dom-expected.txt

```diff
@@ -0,0 +1,9 @@
+This test ensures we can access content inside a deep DOM without crashing.
+
+PASS: webArea.childAtIndex(0).role.toLowerCase().includes('button') === true
+
+PASS successfullyParsed is true
+
+TEST COMPLETE
+Press
+
```

Comment:
Can we construct this deeply nested DOM structure in script instead of checking in a large file with hard-coded tags?